### DISCUSSION TITLE
iterm顶端问好

### DISCUSSION THREAD

**Discussion by WzyPaopao** [AGou-ops/dotfiles#3] [Q&A] [answered]
您好，按照您的配置后，iterm打开vim后，顶部不是x而是问号，请问该怎么处理呢
<img width="946" alt="image" src="https://user-images.githubusercontent.com/34997244/147830675-3f77d091-10f2-41c3-a028-8cd6dd907e0f.png">

**Comment by AGou-ops** [comment, marked as answer]
@WzyPaopao 靓仔你好，这个是需要安装字体的（内置图标的字体），你可以去这个仓库 [nerd-fonts](https://github.com/ryanoasis/nerd-fonts) 里下载一个你喜欢的字体，或者自己patch一个字体包，然后换上就显示正常了.

**Comment by WzyPaopao** [comment]
谢谢靓仔！哈哈哈
